### ISSUE TITLE
Glyph of छ in consonant clusters in Sanskrit font

### ISSUE THREAD

**Issue by sridatta1** [TiroTypeworks/Indigo#32]
The ligature involving ch as in च्छ्ल,  छ्व the vertical stem is removed and upper portion of the conjunct is touching the headstroke
![image](https://user-images.githubusercontent.com/23263723/180656685-aa621777-7b08-4c07-b7b6-d996fb17bd11.png)

I haven't seen similar design in other fonts
![image](https://user-images.githubusercontent.com/23263723/180656795-7ddaa9c5-5db5-47db-8427-04e0c15b11bc.png)
![image](https://user-images.githubusercontent.com/23263723/180656833-8072beac-bd87-4273-869b-29fbc4ae384a.png)

Design in Sanskrit2003 font
![image](https://user-images.githubusercontent.com/23263723/180656915-4fa55e00-7693-4b12-be3f-520e59dad1f8.png)

![image](https://user-images.githubusercontent.com/23263723/180656948-eb4c6cc8-b055-4654-b894-86dfed57843c.png)

![image](https://user-images.githubusercontent.com/23263723/180656964-3b39d576-b95e-4f1c-aaf3-374700d62420.png)

Some fonts also remove the stems in छ्य or use the rest as half froms ( as in Noto forms) which also looks incorrect to me I haven't seen them in printed texts on Sanskrit2003
![image](https://user-images.githubusercontent.com/23263723/180657101-c9f261df-5ef6-4a24-9761-809b6662a179.png)
From https://archive.org/details/SanskritAlphabetsPrimerVenkataramaSastriS/page/n15/mode/1up?view=theater, this book also has useful examples for Sanskrit Conjuncts

**Comment by tiroj** [comment]
This is by design. I think it is something that can be traced back to Linotype Devanagari, and hence perhaps to some older foundry types? My approach in the Tiro font is that the form without the connecting stem becomes the half-form; te vertically compressed form that merges into the headstroke is used when there is a subscript form (thinking about that in the context of Linotype Devanagari, perhaps that design originated in newspaper use?).

**Comment by sridatta1** [comment]
NiWould be interesting to the origin in older type foundries or if any other fonts have this design.
On this query, checked in Nirnaya Sagar press books, didn't see this form.
![Screenshot_20220724-231334__01](https://user-images.githubusercontent.com/23263723/180919906-002b4fff-adce-4821-8094-d37667e77efe.jpg)
So perhaps traced to other type foundries

**Comment by tiroj** [comment]
Considering changing this in Sanskrit font—and possibly in Marathi—but retaining in Hindi font. Reasoning is that the Hindi generally favours more vertically compressed or horizontal forms.

**Comment by sridatta1** [comment]
I checked in the new Linotype Devanagari font here
https://www.myfonts.com/collections/linotype-devanagari-font-monotype-imaging
The vertically compressed forms with the line removed for छ्व च्छ्व are not found
![image](https://user-images.githubusercontent.com/23263723/193462184-14fca9f5-875f-4e12-8125-3a3d999ac221.png)
Don't know about the original designs for this typeface

**Comment by tiroj** [comment]
New forms for the Sanskrit and Marathi fonts, to be implemented in next update.
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/5614500/193465855-75d2380c-c40e-4386-a08a-d939cd5a85e0.png">

**Comment by tiroj** [comment]
Now testable with `v160beta.yml` build path.

**Comment by sridatta1** [comment]
Will the current glyph of छ्य ( and the clusters ending with it) be retained in the Sanskrit font?

**Comment by tiroj** [comment]
Yes. I think the form of छ disconnected from the headline reads more readily as a half-form.
